flow/otel_metrics/otel_manager.go

```diff
@@ -18,15 +18,19 @@ import (
 )
 
 const (
-	SlotLagGaugeName                    string = "cdc_slot_lag"
-	OpenConnectionsGaugeName            string = "open_connections"
-	OpenReplicationConnectionsGaugeName string = "open_replication_connections"
-	IntervalSinceLastNormalizeGaugeName string = "interval_since_last_normalize"
-	FetchedBytesCounterName             string = "fetched_bytes"
+	SlotLagGaugeName                    = "cdc_slot_lag"
+	CurrentBatchIdGaugeName             = "current_batch_id"
```

Comment:
Maybe we should call it sync_batch_id

Also wondering if we should emit this twice, one at start and one at end, can attach different labels for both the times.